less/_components/events/eclipsefdn-user-display.less

```diff
@@ -61,6 +61,7 @@
 .eclipsefdn-user-display-content {
   // make sure the avatars/images inside eclipsefdn-user-display-content will have the same size
   .img-responsive {
-    width: 100%;
+    max-height: 200px;
+    margin: 0 auto; // to make the images horizontally centered in the container
```

Comment:
we shouldn't have comments in here when possible, as they just muddy up the file

Reply:
Ah, OK, I will add a variable for the max-height and remove the comments

Reply:
Comments should be removed when we compile the less file into css. So I would argue that it's encouraged.

Reply:
> Comments should be removed when we compile the less file into css. So I would argue that it's encouraged.

Oh.., OK.
Em..., the comments here are trying to explain some relatively obvious things, so maybe we can remove them for this case? This can save a little bit space because the less file will also be included in the dependency (node_modules).